src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/assets/LicenseScanTests/LicenseExclusions.txt

```diff
@@ -242,6 +248,7 @@ src/vstest/src/package/Microsoft.VisualStudio.TestTools.TestPlatform.V2.CLI/Lice
 
 # Not applicable to source
 src/wpf/src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Media/FontEmbeddingRights.cs|proprietary-license
+src/wpf/src/Microsoft.DotNet.Wpf/src/WindowsBase/MS/Internal/IO/Packaging/CompoundFile/RightsManagementEncryptionTransform.cs|proprietary-license
```

Comment:
I think this belongs in the false positive section. We should also remove the `https://github.com/dotnet/source-build/issues/4373` link there as it adds no value.